File: tuts/005-cloudfront-gettingstarted/swift/Sources/GettingStartedScenario.swift
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import AWSCloudFront

@main
struct GettingStartedScenario {
    static func main() async throws {
        let client = try CloudFrontClient(region: "us-east-1")
        let wrapper = CloudFrontWrapper(client: client)
        print("Running CloudFront getting started scenario...")
        // TODO: setup, interact, teardown
        let _ = wrapper
    }
}

File: tuts/005-cloudfront-gettingstarted/php/GettingStartedScenario.php
<?php
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

require 'vendor/autoload.php';

use Aws\CloudFront\CloudFrontClient;

$client = new CloudFrontClient(['region' => 'us-east-1']);
echo "Running CloudFront getting started scenario...\n";
// TODO: setup, interact, teardown
echo "Scenario complete.\n";
